src/moonlink/src/storage/mooncake_table.rs

```diff
@@ -446,6 +446,9 @@ pub struct MooncakeTable {
 
     /// WAL manager, used to persist WAL events.
     wal_manager: WalManager,
+
+    /// LSN of pending flushes.
+    pub pending_flush_lsns: BTreeSet<u64>,
```

Comment:
`pending` means not yet decided: https://www.merriam-webster.com/dictionary/pending
I'm not sure if it means (1) ongoing, (2) in flush queue, if we have one; (3) in-memory, will be flushed.
```suggestion
    pub ongoing_flush_lsns: BTreeSet<u64>,
```

Reply:
changed